app/src/main/res/values/strings.xml

```diff
@@ -610,6 +610,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="media_detail_step_title">Media Details</string>
   <string name="menu_view_category_page">View category page</string>
   <string name="menu_view_item_page">View item page</string>
+  <string name="default_app_language">Default app language</string>
```

Comment:
Please make it `App user interface language` (and also modify the variable names accordingly).